### PR TITLE
flag also TOF noisy channel in diagnostic

### PR DESCRIPTION
**Pull request by noferini**

### CODE REVIEW

**Comment by noferini** [comment]
Hi @chiarazampolli,
I added also noisy channels flag to diagnostic.
This will guarantee that we will discover them also in the sync phase in case we miss to switch off some of them in the slow control

File: DataFormats/Detectors/TOF/include/DataFormatsTOF/Diagnostic.h [hunk 1]
```diff
@@ -36,9 +36,11 @@ class Diagnostic
   int getFrequency(ULong64_t pattern);                                                    // Get frequency
   int getFrequencyROW() { return getFrequency(0); }                                       // Readout window frequency
   int getFrequencyEmptyCrate(int crate) { return getFrequency(getEmptyCrateKey(crate)); } // empty crate frequency
+  int fillNoisy(int channel, int frequency = 1) { return fill(getNoisyChannel(channel), frequency); }
   int fillROW() { return fill(0); }
   int fillEmptyCrate(int crate, int frequency = 1) { return fill(getEmptyCrateKey(crate), frequency); }
   ULong64_t getEmptyCrateKey(int crate);
+  ULong64_t getNoisyChannel(int channel);
```
Comment (shahor02) [inline DataFormats/Detectors/TOF/include/DataFormatsTOF/Diagnostic.h]: isn't this const method?

Comment (noferini) [reply DataFormats/Detectors/TOF/include/DataFormatsTOF/Diagnostic.h]: Yes, I will fix it (the same for getEmptyrateKey)
Even better -> static

**Comment by chiarazampolli** [comment]
Thanks, it should be transparent for the diagnostic calibration, unless I overlooked something.

**Comment by noferini** [comment]
> Thanks, it should be transparent for the diagnostic calibration, unless I overlooked something.

Yes, it is transparent

**Comment by noferini** [comment]
all checks passed!

**Review by shahor02 — APPROVED** [review]

**Pull request merged**